test/aave-v3/TestSupplyVault.t.sol

```diff
@@ -490,27 +490,100 @@ contract TestSupplyVault is TestSetupVaults {
         assertEq(token.balanceOf(_to), _toTransfer);
     }
 
-    function testAccrueRewardsToCorrectUser() public {
-        uint256 amount = 1e6 ether;
+    function testRewardsShouldAccrueWhenDepositingOnBehalf() public {
+        uint256 amount = 10_000 ether;
+        address[] memory poolTokens = new address[](1);
+        poolTokens[0] = aDai;
 
-        vm.startPrank(address(supplier1));
-        ERC20(dai).approve(address(daiSupplyVault), type(uint256).max);
-        daiSupplyVault.deposit(amount, address(supplier1));
+        vaultSupplier1.depositVault(daiSupplyVault, amount);
+        vm.roll(block.number + 100);
```

Comment:
We should use warp for Aave

I think this is testing nothing as `userReward` or `expectedTotalRewardsAmount` may be close to 0 because of that no?

Can we add a test `assertGt` to?